src/ontoweaver/tabular.py

```diff
@@ -117,26 +117,18 @@ def source_id(self, i, row):
 
 
     def properties(self, row, type):
-        """Extract properties of `type` from `row`."""
+        """Extract properties of each property category for the given node type. If no properties are found, return an empty dictionary."""
+
         properties = {}
-        # FIXME HERE properties for source node are not pulled.
-
-        # Find first matching parent class.
-        matching_class = None
-        for parent in type.mro(): # mro is guaranted in resolution order.
-            if parent in self.properties_of:
-                matching_class = parent
-                break
-        if not matching_class:
-            return {} # Defaults to no property.
-
-        # Extract and map the values.
-        for in_prop in self.properties_of[matching_class]:
-            out_prop = self.properties_of[type.__name__][in_prop]
-            properties[out_prop] = row[in_prop]
 
-        return properties
+        if type.__name__ in self.properties_of:
```

Comment:
The previous implementation was able to find properties that were defined for super-classes. Is this new implementation doing that as well?

The current implementation of properties is to store them as strings attached in a single member of each class. This should more favorably follow the expected design of using actual member variables, which would have the advantage of bringing automatic inheritance of member variables. However, this may be more difficult to implement (how to collect all properties for iteration? Name collision? Should we use a prefix for member variables that are properties? How to enforce that properties should be strings? At input or output? etc.).